### ISSUE TITLE
DFT not faster with chunked xarrays

### ISSUE THREAD

**Issue by tomchor** [xgcm/xrft#74]
Maybe I'm missing something, but applying a DFT on an xarray chunked with dask is not faster than a regular xarray. In the README it is mentioned that "uses the dask API of FFT and map_blocks to allow parallelization of DFT", so I expected it to be faster.

Here's an illustration of what I mean

```
In [4]: θ                                  
Out[4]: 
<xarray.DataArray (itime: 10, x: 320, y: 320, z: 144)>
array([[[[283.121695, ..., 282.878798],
         ...,
         [283.120745, ..., 282.878393]],

        ...,

        [[283.10938 , ..., 282.876364],
         ...,
         [283.117299, ..., 282.874488]]],


       ...,


       [[[283.104731, ..., 282.878724],
         ...,
         [283.110358, ..., 282.874231]],

        ...,

        [[283.093765, ..., 282.873418],
         ...,
         [283.115609, ..., 282.873015]]]])
Coordinates:
  * itime    (itime) int64 95500 96000 96500 97000 ... 98500 99000 99500 100000
  * x        (x) float64 0.625 1.875 3.125 4.375 ... 395.6 396.9 398.1 399.4
  * y        (y) float64 0.625 1.875 3.125 4.375 ... 395.6 396.9 398.1 399.4
  * z        (z) float64 -0.3125 -0.9375 -1.562 -2.188 ... -88.44 -89.06 -89.69

In [6]: %time a=xrft.dft(θ, dim="x")       
CPU times: user 43.2 s, sys: 1min 5s, total: 1min 48s
Wall time: 10.7 s

In [7]: pθ = θ.chunk(dict(itime=2))        

In [8]: %time a=xrft.dft(pθ, dim="x").compute()                                       
CPU times: user 17.3 s, sys: 3min 12s, total: 3min 29s
Wall time: 12.7 s

```

So DFTing an xarray distributed over 5 chunks is as fast as (a bit slower even) doing it with a regular array. That seems weird to me. Am I missing something obvious here?

PS.: 
`xrft` version 0.2.0 
`xarray` version 0.12.1
`dask` version 1.2.0

**Comment by rabernat** [comment]
What kind of computer are you using? How many cores does it have?

**Comment by tomchor** [comment]
I'm using a mini-cluster with CentOS. It has 28 physical cores. 

Your question made me consider the possibility that dask is not able to operate properly on my machine. I'm very unexperienced on `dask`, so I'm not sure how to diagnose if there's a problem. Based on the fact that I have 5 chunks, I tried issuing the command below (based on [this answer](https://stackoverflow.com/questions/51285420/dask-specify-number-of-processes)) before the parallel stuff:

```
import dask
from multiprocessing.pool import Pool
dask.config.set(pool=Pool(5))
```

as a result the command `%time a=xrft.dft(pθ, dim="x").compute()` hung there for at least a good minute before I got tired and killed it.
